exercises/03-sagas/README.md

```diff
@@ -143,12 +143,11 @@ Make sure everything compiles.
 
 ### Step 5
 
-Store the `CustomerId`, `OrderId` and products in the state of the saga. We can do this by accessing the saga's `Data` property, as in `Data.CustomerId`. Set the appropriate properties in both `Handle` methods.
+Store the `CustomerId` and products in the state of the saga. We can do this by accessing the saga's `Data` property, as in `Data.CustomerId`. Set the appropriate properties in both `Handle` methods.
```

Comment:
Do you want to mention that as the correlation id, OrderId is already prefilled in the object as well? (For clarity?)

Reply:
Yeah, I was considering this too. I'll add something.